### PR TITLE
Remove flow group id field

### PR DESCRIPTION
**Pull request by dpoltavets**
**MERGE NOTE: This PR should only be merged after #4404 is released!**

### CODE REVIEW

**Review by sergii-iakovenko — APPROVED** [review]

**Review by niksv — APPROVED** [review]

File: docker/db-migration/migrations/013-remove-flow-group-id-field.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
```
Comment (surabujin) [inline docker/db-migration/migrations/013-remove-flow-group-id-field.yaml:1]: Why we have this file?

**Review by surabujin — CHANGES_REQUESTED** [review]

File: docker/db-migration/migrations/014-remove-flow-group-id-field.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      author: dpoltavets
+      changes:
+        - tagDatabase:
+            tag: 014-remove-flow-group-id-field
+
+  - changeSet:
+      id: remove_flow_group_id_field
+      author: dpoltavets
+      changes:
+        - sql: "DROP INDEX flow.group_id"
```
Comment (surabujin) [inline docker/db-migration/migrations/014-remove-flow-group-id-field.yaml:13]: We must delete fileds itself (i.e. `UPDATE flow REMOVE group_id`).

**Review by surabujin — CHANGES_REQUESTED** [review]

File: docker/db-migration/migrations/013-remove-flow-group-id-field.yaml (hunk 1 above)
Comment (dpoltavets) [reply docker/db-migration/migrations/013-remove-flow-group-id-field.yaml]: Rebase artifact. This file has been deleted.

File: docker/db-migration/migrations/014-remove-flow-group-id-field.yaml (hunk 2 above)
Comment (dpoltavets) [reply docker/db-migration/migrations/014-remove-flow-group-id-field.yaml:13]: This command has been added

**Review by timofei-durakov — APPROVED** [review]

**Pull request merged**